app/models/item.rb

```diff
@@ -37,4 +37,18 @@ def no_orders?
   def quantity_purchased
     item_orders.sum(:quantity)
   end
+
+  def quantity_ordered(item_id)
+    ItemOrder.where("item_id = ?", "#{item_id}").sum(:quantity)
+  end
+
+  def fulfilled?(order_id)
+    return false if item_orders.where(order_id: order_id).first.fulfill_status == 'unfulfilled'
```

Comment:
I think this can be refactored to: 
`item_orders.where(order_id: order_id).first.fulfill_status == 'unfulfilled'`

Reply:
Good catch - refactored this as suggested